src/DIRAC/FrameworkSystem/DB/ComponentMonitoringDB.py

```diff
@@ -382,9 +382,12 @@ def getComponentsStatus(self, conditionDict={}):
             if self.__componentMatchesCondition(compDict, requiredComponents, conditionDict):
               statusSet.addUniqueToSet(requiredComponents, compDict)
         # Walk the URLs
-        serviceURLs = getSystemURLs(system, setup)  # verify URLs in getSystemURLs method
-        for service in serviceURLs:
-          url = urlparse.urlparse(serviceURLs[service])
+        systemURLs = getSystemURLs(system, setup)  # verify URLs in getSystemURLs method
+        for service in systemURLs:
+          # systemURLs is a dict that contain a list of URLs for service
+          if not systemURLs[service]:
+            return S_ERROR('Not found URL for %s service.' % service)
```

Comment:
Are you sure about returning an error here? Maybe just printing it out and keep going?

Reply:
fixed